### ISSUE TITLE
macOS Sierra & Include Media

### ISSUE THREAD

**Issue by chillybin** [cdukes/bones-for-genesis-2-0#31]
Getting this error when trying to compile on macOS Sierra

```
Running "sass:build" (sass) task
Error: Invalid US-ASCII character "\xE2"
        on line 263 of bower_components/include-media/dist/_include-media.scss
        from line 5 of sass/style.scss
  Use --trace for backtrace.
Warning: Exited with error code 65 Use --force to continue.

Aborted due to warnings.

Process finished with exit code 6
```

**Comment by cdukes** [comment]
@chillybin 

Managed to fix this and #30? Was it a local configuration issue, or something I should update in the repo?

**Comment by chillybin** [comment]
No sorry I figured they're weren't really your issue so logged jobs under the scripts. But yeah having both issues still

**Comment by chillybin** [comment]
https://github.com/eduardoboucas/include-media/issues/84#issuecomment-246679191

**Comment by chillybin** [comment]
Looks like @charset "UTF-8"; solves it on all SCSS files

**Comment by cdukes** [comment]
Hmm, I'll look at this once Sierra is out. The ticket you referenced is talking about Ruby defaults, but we're using the [C++ compiler](https://github.com/sindresorhus/grunt-sass). Could be the same issue though

**Comment by chillybin** [comment]
Teaches me for using the GM build, I still have to do some more troubleshooting as it might be a local issue. I was setting @charset "UTF-8"; but one of the grunt tasks was setting it to @charset 'UTF-8'; which was making it invalid. I also had to remove all the ligatured operators.

I pulled down your version from September 7 and that seemed to compile with no issues but i didnt add test any changes, just ran the default grunt task.
